onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -54,11 +56,88 @@ ORT_API_STATUS_IMPL(OrtApis::InvokeOp,
 ORT_API(void, OrtApis::ReleaseOp, _Frees_ptr_opt_ OrtOp*) {
 }
 
+ORT_API_STATUS_IMPL(OrtApis::CopyKernelInfo,
+                    _In_ const OrtKernelInfo*,
+                    _Outptr_ OrtKernelInfo**) {
+  API_IMPL_BEGIN
+  return CreateStatus(ORT_NOT_IMPLEMENTED, "CopyKernelInfo is not implemented for minimal build.");
+  API_IMPL_END
+}
+
+ORT_API(void, OrtApis::ReleaseKernelInfo, _Frees_ptr_opt_ OrtKernelInfo*) {
+}
+
 #else
 
 namespace onnxruntime {
 namespace standalone {
 
+void ReleaseNode(onnxruntime::Node* node) {
+  if (node) {
+    for (auto* input_arg : node->InputDefs()) {
+      delete input_arg;
+    }
+    for (auto* output_arg : node->OutputDefs()) {
+      delete output_arg;
+    }
+    delete node;
+  }
+}
+
+using NodePtr = std::unique_ptr<onnxruntime::Node, void (*)(onnxruntime::Node*)>;
+using StandAloneNodesMap = InlinedHashMap<const onnxruntime::OpKernel*, NodePtr>;
+
+class NodeRepo {
+ private:
+  NodeRepo() = default;
+  ~NodeRepo() = default;
+
+  ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(NodeRepo);
+
+  static std::mutex kMtx;
```

Comment:
Why static? Can be just a regular member, no?

Reply:
yes - let's change to regular.